src/newgame.js

```diff
@@ -0,0 +1,20 @@
+const fetch = require("node-fetch");
+
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+const leaderBoard = () => {
+    fetch(baseURL, {
+      method: 'POST',
+      body: JSON.stringify({
+        name: 'Strikers Game',
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json())
+      .then((json) => json)
+      .catch((error) => console.error('Error:', error));// eslint-disable-line
```

Comment:
- [ ] Kindly try to avoid pushing the console statement as per best code practices.

Reply:
## @hemant-soni-vst-au4 Thank you! 🤝 
- I have taken note and will treat all optional changes ASAP. 😄 
- Also will fix the link error 🙏